### ISSUE TITLE
Attempting to use a modified version with latest lwjgl, error

### ISSUE THREAD

**Issue by kode54** [yusefnapora/m1-multimc-hack#7]
"Cocoa: Regular windows do not have icons on macOS."

The relevant line in GLFW is over 14 months old, so it should have affected your build as well.

**Comment by MoustacheOff** [comment]
> "Cocoa: Regular windows do not have icons on macOS."
> 
> The relevant line in GLFW is over 14 months old, so it should have affected your build as well.

@kode54 Are you using a fork of this repo?

**Comment by kode54** [comment]
Yes, and I have a fork, where I had to alter GLFW natives to remove that error message to get it working. I also found that the newer LWJGL only works with Minecraft 1.17 or newer, while the one in this repository works with 1.13.x-1.16.x as well. There are at least a few forks in here that either change it to work with 1.12 or older, or detect that at runtime and provide for both.

**Comment by Blyrexx** [comment]
@kode54 Yesterday I also created a fork of this repository with the latest lwjgl 3.3.0 snapshot, but when I start the game I get this error: "GLFW error 65548: Cocoa: Regular windows do not have icons on macOS". It sounds like you had the same problem. What did you do to fix this error?

**Comment by kode54** [comment]
I had to build my own copy of the relevant object file from GLFW, removing the error from the unsupported API call.

**Comment by Blyrexx** [comment]
Which tools did you use to build your own copy?

**Comment by MoustacheOff** [comment]
This error seems to cause problems to a lot

**Comment by kode54** [comment]
I think I may have forked their entire git repo and modified the source, then ran `brew edit lwjgl` and instructed brew to build a git head copy from my repo. Then again, I don't even remember if brew is capable of building anything any more.

I am infamous as an author of "use HEAD as authority" for my own crap, while at the same time frequently putting other developers' code on permanent version freeze the instant I have the bare minimum of functional working copy.
